### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by Charmis-Square**
Updated Documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,14 +26,14 @@ In addition to this README, the following is available in the [React Native plug
...
-* Android Gradle Plugin: 3.0.0 or greater.
+* Android Target SDK version: API 33 (Android 13).
+* Android Gradle Plugin: 4.0.0 or greater.
 
 ### iOS
 
-* Xcode version: 9.1 or greater.
-* iOS Base SDK: 12.0 or greater.
-* Deployment target: iOS 12.0 or greater.
+* Xcode version: 14.0 or greater.
```
Comment (brandonjenniges) [inline README.md:34]: Can you switch this to Xcode 14.1?  That's the current minimum to be able to upload to the App Store.

File: docs/get-started.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ environment.
 ## Process overview
 
 * [Step 1: Create a React Native project](#step-1-create-a-react-native-project)
-* [Step 2: Install and link the React Native plugin for In-App Payments SDK](#step-2-install-and-link-the-react-native-plugin-for-in-app-payments-sdk)
+* [Step 2: Install the React Native plugin for In-App Payments SDK](#step-2-install-and-link-the-react-native-plugin-for-in-app-payments-sdk)
```
Comment (brandonjenniges) [inline docs/get-started.md:22]: I think this link doesn't work anymore.  

Should it be `#step-2-install-the-react-native-plugin-for-in-app-payments-sdk`?

File: README.md (hunk 1 above)
Comment (Charmis-Square) [reply README.md:34]: Done change

Comment (brandonjenniges) [reply README.md:34]: Sorry, my wording was a little unclear.  But it should be:

```
Xcode version: 14.1 or greater.
iOS Base SDK: 13.0 or greater.
Deployment target: iOS 13.0 or greater.
```

Comment (Charmis-Square) [reply README.md:34]: ohh sorry it's done

**Review by Armaxis — APPROVED** [review]

**Pull request merged**